### PR TITLE
Todo list: Interactive list

### PR DESCRIPTION
**Pull request by alvarorf**
In this pull request, the following features are added:
- A new Javascript file ('status.js') contains the functionality about the status's updates, via the 'toggle()' function.
- The checkbox for each task has an event listener.
- A function has been implemented for the button 'Clear all completed' ('clearCompleted()').
- The updated array of items is stored in local storage, so the user gets the correct list values after the page reloads.
- When the user tries to edit a task, he/she should press enter to confirm the edit (if he/she focuses out, it is understood that he/she cancelled the edit and the changes won't be applied. This is the expected behavior).

Thank you for your comments.

### CODE REVIEW

File: src/crud.js [hunk 1]
```diff
@@ -1,3 +1,5 @@
+import toggle from './status.js';
+
 class Crud {
   static addTodo() {
     const todos = Crud.retrieveTodos();
```
Comment (AmaduKamara) [inline src/crud.js:5]: Currently, all the functionalities are working in your app but there are a few changes you need to fix.

- [ ] The add todo is working fine but when I try to enter multiple words in the input field by clicking the space bar key on the keyboard, the todo gets submitted instantly without allowing me to enter other words. I believe a todo is not just one word and it must only get submitted when I click the enter key on the keyboard or a submit button(in this case there is no submit button for this design. That's ok, no worries.). 
- [ ] Kindly make it possible for a user to enter multiply words for todo and only the enter key on the keyboard gets the multiple texts submitted 👍 
- [ ] Also, please do the same fix for updating the todo, I noticed the same problem when I try to update a todo, it does not allow me to enter multiple words. Please fix these issues and your project will be great. 👍

**Review by AmaduKamara — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️♻️♻️ 

Hi @alvarorf,

AlonzoLeroneThisCloseGIF

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✅ Correct Gitflow setup
✅ No linter error
✅ README is professional
✅ Good descriptive PR


## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/crud.js [hunk 2]
```diff
@@ -104,15 +108,6 @@ class Crud {
...
-    const todos = Crud.retrieveTodos();
-    todos[parseInt(index, 10)].completed = !todos[parseInt(index, 10)].completed;
-    Crud.updateTodos(todos);
-    Crud.renderTodos();
-    Crud.refreshEvents();
-  };
-
   static enableEdit = (index) => {
     const element = document.getElementById(`data-${index}`);
     element.style.display = 'none';
```
Comment (ShahierNashaat) [inline src/crud.js:113]: - [ ] Kindly be noticed that when I try to edit a task the task doesn't update with the new value please fix that.
- [ ] Also when I enter a task with spaces and try to edit it the other words don't show like in the picture.
Screenshot from 2022-05-06 00-17-43
Screenshot from 2022-05-06 00-18-01

**Review by ShahierNashaat — CHANGES_REQUESTED** [review]
Hi @alvarorf , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/crud.js (hunk 2 above)
Comment (alvarorf) [reply src/crud.js:113]: Thank you for pointing out this issue. Now, when the user tries to edit a task, he/she should press enter to confirm the edit (if he/she focuses out, it is expected that he/she cancelled the edit and the changes won't be applied. This is the expected behavior). It has now been solved. Thanks again.

**Review by brytebee — APPROVED** [review]
Hi, @alvarorf👋,

# STATUS: APPROVED 🕺🥳 

Congratulations🍾🥳!! Your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 



Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**